src/kernel/chainparams.cpp

```diff
@@ -191,6 +191,12 @@ class CMainParams : public CChainParams {
                 .hash_serialized = AssumeutxoHash{uint256{"a2a5521b1b5ab65f67818e5e8eccabb7171a517f9e2382208f77687310768f96"}},
                 .m_chain_tx_count = 991032194,
                 .blockhash = consteval_ctor(uint256{"0000000000000000000320283a032748cef8227873ff4872689bf23f1cda83a5"}),
+            },
+            {
+                .height = 880'000,
+                .hash_serialized = AssumeutxoHash{uint256{"dbd190983eaf433ef7c15f78a278ae42c00ef52e0fd2a54953782175fbadcea9"}},
+                .m_chain_tx_count = 1145604538,
+                .blockhash = consteval_ctor(uint256{"000000000000000000010b17283c3c400507969a9c2afd1dcf2082ec5cca2880"}),
```

Comment:
Now that https://developercommunity.visualstudio.com/t/consteval-conversion-function-fails/1579014#T-N10721450 was released, I'm wondering if we can simplify this to:
```suggestion
                .blockhash = uint256{"000000000000000000010b17283c3c400507969a9c2afd1dcf2082ec5cca2880"},
```
or even add an implicit `constexpr uint256(const std::array<std::byte, 32>& hex) : base_blob(hex) {}` and do:
```suggestion
                .blockhash = "000000000000000000010b17283c3c400507969a9c2afd1dcf2082ec5cca2880"_hex,
```

-------

I'll investigate and open a follow-up PR if there are no strong objections.